src/services/destination/nativeIntegration.ts

```diff
@@ -63,23 +64,30 @@ export class NativeIntegrationDestinationService implements DestinationService {
     const destHandler = FetchHandler.getDestHandler(destinationType, version);
     const respList: ProcessorTransformationResponse[][] = await Promise.all(
       events.map(async (event) => {
+        const metaTO = this.getTags(
+          destinationType,
+          event.metadata?.destinationId,
+          event.metadata?.workspaceId,
+          tags.FEATURES.PROCESSOR,
+        );
+        metaTO.metadata = event.metadata;
+        const metadataWithSvcCtx = {
```

Comment:
I thought we can simply add to requestMetadata like following:
requestMetadata.context = {...};

Reply:
We are getting an eslint error when we do as suggested hence adopted this approach